src/graphql/operations/users.ts

```diff
@@ -26,20 +28,41 @@ export default async function (parent, args) {
 
   const query = `
     SELECT
-      u.*,
-      SUM(l.vote_count) as votesCount,
-      SUM(l.proposal_count) as proposalsCount,
-      MAX(l.last_vote) as lastVote
+      u.*
```

Comment:
Maybe keep the leaderboard `JOIN` on this query ?

So that if someone requests an user present in both users and leaderboard tables, we can do everything with this single query.

Reserve the second query only in case address exists only on leaderboard table, and not users.

Reply:
Changed it

Reply:
The `INNER JOIN` will only consider addresses that exists in both users and leaderboard table.

Better to user `LEFT JOIN`, so we catch addresses that exists on both tables and only on users table on the first query (with some `COALESCE` on the counters), so the second query will take care only of addresses that do not exist at all on the users table.